### DISCUSSION TITLE
Welcome to pipedal Discussions!

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#35] [Releases]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by joseaides** [comment]
Looper question:
Currently you can create a loop by using the looper plugin in a preset, but if then you cnahge the preset, the loop is gone.

Is there  any way to create loops and use them with other presets?

Thanks in advance
